src/MSALWrapper.Benchmark/BrokerBenchmark.cs

```diff
@@ -89,7 +89,7 @@ private IPCAWrapper BuildPCAWrapper(ILogger logger, Guid clientId, Guid tenantId
 ;
             if (useNativeBroker)
             {
-                clientBuilder.WithBrokerPreview();
+                clientBuilder.WithBroker(new BrokerOptions(BrokerOptions.OperatingSystems.Windows));

```

Comment:
@goagain, would you be able to summarize the changes that you made in a comment on this PR? I can guess why you made some of these changes, but it would help if you commented on why this isn't just a straightforward version bump.

Reply:
We introduced native broker mode in MSAL 4.43. At that time, Native broker is provided as a preview method.

Because we are bumping to the newest MSAL, the API `WithBrokerPreview` is marked as deprecated, and now `clientBuilder.WithBroker(new BrokerOptions(BrokerOptions.OperatingSystems.Windows))` is the suggested way.